drivers/java/driver.go

```diff
@@ -97,9 +97,9 @@ func init() {
 // TaskConfig is the driver configuration of a taskConfig within a job
 type TaskConfig struct {
 	Class     string   `codec:"class"`
-	ClassPath string   `codec:"classpath"`
+	ClassPath string   `codec:"class_path"`
```

Comment:
Nit: Java classpath is traditionally spelled as a single word so lets use:
```
  Classpath string `codec:"classpath"`
```

Reply:
we can't change what the current driver takes though? https://www.nomadproject.io/docs/drivers/java.html#class_path

Reply:
Yeah, let's not change things without also implementing and testing the upgrade path.

Reply:
Yeah - I mistakenly went with `classpath` for the reason you mentioned, but then realized that existing drivers use `class_path` and didn't want to break compatbility

Reply:
Ahhhhhh okay!